azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-connector-examples.adoc

```diff
@@ -27,7 +27,7 @@ image::azure-sb-mgmt-flow-get-queue.png[This app flow shows the components used
 The *Get Entity* operation returns information about the specified queue, including configuration information and information about the metadata:
 
 . In the *Mule Palette* view, search for *azure* and select the Azure Service Bus Management Connector *Get Entity* operation.
```

Comment:
All of the "search for" texts should be formatted as syntax (` `) rather than bold (* *). 
Please search for and replace.